sda-commons-server-mongo-testing/src/main/java/org/sdase/commons/server/mongo/testing/UseExistingMongoDb.java

```diff
@@ -75,14 +46,15 @@ public String getServerVersion() {
 
   protected void logConnection() {
     if (LOG.isInfoEnabled()) {
-      String withPasswordText = isNotBlank(password) ? "using a password" : "without password";
+      String withPasswordText =
+          mongoConnectionString.getPassword() != null ? "using a password" : "without password";
       LOG.info(
           "Testing as '{}' {} in MongoDB '{}' at '{}' with options '{}'",
-          username,
+          mongoConnectionString.getUsername(),
           withPasswordText,
-          getDatabase(),
-          getHosts(),
-          getOptions());
+          mongoConnectionString.getDatabase(),
+          mongoConnectionString.getHosts(),
+          optionsFromMongoUrl(connectionString));
```

Comment:
Could we keep onlu mongoConnectionString variable and instead of the `connectionString` variable we use the `mongoConnectionString.getConnectionString()` method?

Reply:
Yes, that was possible. 👍 
I've updated `UseExistingMongoDb` and `StartLocalMongoDb`.